src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -204,7 +205,7 @@ const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: '#828282',
+    color: COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous ones, replacing hard-coded color values with dynamic ones from a theme object. This is a good practice.
```diff
-    color: '#828282',
+    color: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->